.github/workflows/_sitrep_mgmn.yaml

```diff
@@ -142,10 +142,20 @@ jobs:
 
           echo "STATUS=${status}" >> ${GITHUB_OUTPUT}
 
+      - name: Check and display metrics summary
```

Comment:
I'm a bit puzzled because it seems like this .json file is not JSON?
https://github.com/NVIDIA/JAX-Toolbox/actions/runs/8917232922/job/24497043508?pr=712#step:7:1

Reply:
Hmm yea that is clearly a markdown file (also I have a typo, metric_summary.json instead of metrics_summary.json).

I made an update to write both a markdown and json file.